### PR TITLE
Replace relationships viewer with new visualization

### PR DESCRIPTION
**Pull request by AustinMoninger**

### CODE REVIEW

File: src/containers/AssetViewer.tsx [hunk 1]
```diff
@@ -13,14 +13,15 @@ import {
 } from '../modules/assetmappings';
 import { RootState } from '../reducers/index';
 import { selectThreeD, ThreeDState } from '../modules/threed';
-import RelationshipNetworkViewer from './RelationshipNetworkViewer';
+// import RelationshipNetworkViewer from './RelationshipNetworkViewer';
```
Comment (anderscognite) [inline src/containers/AssetViewer.tsx]: Do we need this?

**Comment by anderscognite** [comment]
Ahh conflicts

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-33.discovery.preview.cogniteapp.com)

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-33.discovery.preview.cogniteapp.com)

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-33.discovery.preview.cogniteapp.com)

**Pull request merged**